### PR TITLE
Feature/pr electrical windowcovering

### PR DESCRIPTION
**Pull request by TedTolboom**

### CODE REVIEW

File: lib/system/capabilities/measure_power/electricalMeasurement.js [hunk 1]
```diff
@@ -3,20 +3,26 @@
 /**
  * Cluster capability configuration for `measure_power`.
  * @type {ClusterCapabilityConfiguration}
+ * Add below code to your device driver to read the attributes and define the correct formatting
+ * if (typeof this.activePowerFactor !== 'number') {
+ *  const { acPowerMultiplier } = await zclNode.endpoints[this.getClusterEndpoint(CLUSTER.ELECTRICAL_MEASUREMENT)].clusters[CLUSTER.ELECTRICAL_MEASUREMENT.NAME].readAttributes('acPowerMultiplier');
+ *  const { acPowerDivisor } = await zclNode.endpoints[this.getClusterEndpoint(CLUSTER.ELECTRICAL_MEASUREMENT)].clusters[CLUSTER.ELECTRICAL_MEASUREMENT.NAME].readAttributes('acPowerDivisor');
```
Comment (RobinBol) [inline lib/system/capabilities/measure_power/electricalMeasurement.js:9]: Why not combine these two into one call?

File: lib/system/capabilities/meter_power/metering.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+/**
+ * Cluster capability configuration for `measure_power`.
+ * @type {ClusterCapabilityConfiguration}
+ * Add below code to your device driver to read the attributes and define the correct formatting 
+ * if (typeof this.meteringFactor !== 'number') {
+ *   const { multiplier } = await zclNode.endpoints[this.getClusterEndpoint(CLUSTER.METERING)].clusters[CLUSTER.METERING.NAME].readAttributes('multiplier');
+ *   const { divisor } = await zclNode.endpoints[this.getClusterEndpoint(CLUSTER.METERING)].clusters[CLUSTER.METERING.NAME].readAttributes('divisor');
```
Comment (RobinBol) [inline lib/system/capabilities/meter_power/metering.js:9]: Same as above

File: lib/system/capabilities/windowcoverings_state/windowCovering.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  up: {
+    command: 'upOpen',
+  },
+  idle: {
+    command: 'stop',
+  },
+  down: {
+    command: 'downClose',
+  },
+};
```
Comment (RobinBol) [inline lib/system/capabilities/windowcoverings_state/windowCovering.js:14]: Maybe we could flatten this one level:
```
const commandMap = { up: 'upOpen' }
```

**Comment by TedTolboom** [comment]
Fixed the requested changed... Lint fails 2x on len (non critical)

File: lib/system/capabilities/measure_water/flowMeasurement.js [hunk 4]
```diff
@@ -15,6 +15,6 @@ module.exports = {
     // MeasuredValue represents the flow in m3/h as follows:
     // MeasuredValue = 10 x Flow
     // Measure_water represents the flow in l/min = m3/h / 16,6667
-    return Math.round(((value / 10) / 16,6667) * 10) / 10;
+    return Math.round(((value / 10) / 16, 6667) * 10) / 10;
```
Comment (RobinBol) [inline lib/system/capabilities/measure_water/flowMeasurement.js:18]: This can't be right, I think you confused `.` and `,`.

**Comment by TedTolboom** [comment]
Requested change has been fixed; all changes synced to #36

**Pull request merged**